src/main/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/resources/TemporaryAbsencesResource.kt

```diff
@@ -188,7 +190,12 @@ class TemporaryAbsencesResource(
     @PathVariable
     prisonNumber: String,
 
+    @Parameter(description = "The movementId from BASM", required = false)
```

Comment:
meh.. conceptually clients don't need to know that its a BASM ID. If we were to have multiple clients (no plans!) then it would provide us more flexibility to not advertise this and instead encourage clients to treat it transparently. 

(We could change what it is and not break anything thats using it)